### PR TITLE
Teams webhooks

### PR DESCRIPTION
**Pull request by nkramer**

### CODE REVIEW

File: api-reference/beta/api/subscription-post-subscriptions.md [hunk 1]
```diff
@@ -33,13 +35,23 @@ Creating a subscription requires read permission to the resource. For example, t
 |[security alert](../resources/alert.md) | SecurityEvents.ReadWrite.All | Not supported | SecurityEvents.ReadWrite.All |
 |[user](../resources/user.md) | User.Read.All | User.Read.All | User.Read.All |
 
-> **Note:** chatMessage subscriptions require [encryption](/graph/webhooks-with-resource-data). Subscription creation will fail if [encryptionCertificate](../resources/subscription.md) is not specified.
+### chatMessage (Microsoft Teams)
 
-> **Note:** Additional limitations apply for subscriptions on OneDrive and Outlook items. The limitations apply to creating as well as managing (getting, updating, and deleting) subscriptions.
+**chatMessage** subscriptions require [encryption](/graph/webhooks-with-resource-data). Subscription creation will fail if [encryptionCertificate](../resources/subscription.md) is not specified. Before creating a **chatMessage** subscription, you must request access. For details, see [Protected APIs in Microsoft Teams](/graph/teams-protected-apis).
```
Comment (baywet) [inline api-reference/beta/api/subscription-post-subscriptions.md]: Is the access request also required for channel and chat specific endpoints (not the all ones)? We should detail that aspect.

File: api-reference/beta/api/subscription-post-subscriptions.md [hunk 2]
```diff
@@ -22,7 +22,9 @@ Creating a subscription requires read permission to the resource. For example, t
 | Supported resource | Delegated (work or school account) | Delegated (personal Microsoft account) | Application |
 |:-----|:-----|:-----|:-----|
 |[chatMessage](../resources/chatmessage.md) (/teams/{id}/channels/{id}/messages) | Not supported | Not supported | ChannelMessage.Read.All  |
```
[the remark this answers is not in the document]
Comment (nkramer) [reply api-reference/beta/api/subscription-post-subscriptions.md]: No, it's the same permission scope in both cases (all channels or one channel).

File: api-reference/beta/api/subscription-post-subscriptions.md (hunk 1 above)
Comment (baywet) [reply api-reference/beta/api/subscription-post-subscriptions.md]: so in both cases I need to request access before I can use it? (protected APIs)

File: api-reference/beta/api/subscription-post-subscriptions.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (baywet) [reply api-reference/beta/api/subscription-post-subscriptions.md]: so in both cases I need to request access before I can use it? (protected APIs)

[the remark this answers is not in the document]
Comment (nkramer) [reply api-reference/beta/api/subscription-post-subscriptions.md]: Yeah

File: api-reference/beta/api/subscription-post-subscriptions.md (hunk 1 above)
Comment (nkramer) [reply api-reference/beta/api/subscription-post-subscriptions.md]: yes (despite what I said earlier :) )

File: api-reference/beta/api/subscription-post-subscriptions.md [hunk 3]
```diff
@@ -33,13 +35,23 @@ Creating a subscription requires read permission to the resource. For example, t
...
 
-- On personal OneDrive, you can subscribe to the root folder or any subfolder in that drive. On OneDrive for Business, you can subscribe to only the root folder. Notifications are sent for the requested types of changes on the subscribed folder, or any file, folder, or other **driveItem** instances in its hierarchy. You cannot subscribe to **drive** or **driveItem** instances that are not folders, such as individual files.
+> **Note:** `/teams/allMessages` and `/chats/allMessages` are currently in preview. During the preview, you may use this API without fees, subject to the [Microsoft APIs Terms of Use](https://docs.microsoft.com/legal/microsoft-apis/terms-of-use?context=graph/context). However, users of apps that use the API might be required to have subscriptions to specific Microsoft 365 offerings. Upon general availability, Microsoft may require you or your customers to pay additional fees based on the amount of data accessed through the API.
```
Comment (baywet) [inline api-reference/beta/api/subscription-post-subscriptions.md]: @nkramer following the meeting we had the other week, do we want to review this part?

File: concepts/teams-protected-apis.md [hunk 4]
```diff
@@ -15,10 +15,13 @@ The following APIs are currently protected:
 * [List channel messages](/graph/api/channel-list-messages?view=graph-rest-beta) using [application permissions](auth/auth-concepts.md#microsoft-graph-permissions)
 * [Get chatMessages in a channel delta](/graph/api/chatmessage-delta?view=graph-rest-beta) using [application permissions](auth/auth-concepts.md#microsoft-graph-permissions)
 * [Get channel message](/graph/api/channel-get-message?view=graph-rest-beta) using [application permissions](auth/auth-concepts.md#microsoft-graph-permissions)
+* [Create subscription for new channel messages](/graph/api/subscription-post-subscriptions?view=graph-rest-beta) using [application permissions](auth/auth-concepts.md#microsoft-graph-permissions)
 * [List replies to a message](/graph/api/channel-list-messagereplies?view=graph-rest-beta) using [application permissions](auth/auth-concepts.md#microsoft-graph-permissions)
 * [Get a reply to a message](/graph/api/channel-get-messagereply?view=graph-rest-beta) using [application permissions](auth/auth-concepts.md#microsoft-graph-permissions)
+* (List chats using [application permissions](auth/auth-concepts.md#microsoft-graph-permissions) doesn't exist, with or without protective API access)
```
Comment (Lauragra) [inline concepts/teams-protected-apis.md]: Should this say "protected" API?

File: api-reference/beta/api/subscription-post-subscriptions.md (hunk 3 above)
Comment (nkramer) [reply api-reference/beta/api/subscription-post-subscriptions.md]: reviewed & good

**Pull request merged**